### DISCUSSION TITLE
Change osu!Lazer volume wheels to be on top of side-menu.

### DISCUSSION THREAD

**Discussion by Pellepunk04** [ppy/osu#30488] [Ideas] [answered]
As we know, holding down alt and scrolling up/down changes the volume.
![image](https://github.com/user-attachments/assets/928228f0-774b-45f4-8180-b525a3c8aab4)

However, if you have the side-menu open (ctrl+o), you will still change the volume, but the actual volume wheels will be behind the side menu, so you have to close it to see what you're setting your volume at.
![image](https://github.com/user-attachments/assets/23e278f6-ca0d-47b1-bacf-a59be8c6781e)
Bit of a rough screenshot, but you can see the volume wheels getting hidden behind the side-menu.

The reason this should be changed is because the volume wheels are on top of the mod select (F1) menu.
![image](https://github.com/user-attachments/assets/29cf746e-b0cb-4155-8647-ef334f8792c8)

**Comment by Joehuu** [comment, marked as answer]
This is already tracked as an issue in https://github.com/ppy/osu/issues/11077.

**Reply by Pellepunk04** [reply]
I literally searched for a issue containing this but couldn't find one so I made my own, my bad.
